### ISSUE TITLE
Feature request: custom file path for timestamp.json

### ISSUE THREAD

**Issue by RobDesideri** [poppinlp/file-changed#2]
Hi, awesome project! Very useful for my purpose.
One request: a **custom file path** for the timestamp.json?
I see that is set [this](https://github.com/poppinlp/file-changed/blob/master/lib/main.js#L22).
If you want I can do it.

**Comment by poppinlp** [comment]
OK, sure.
PR is very welcome :)

**Comment by RobDesideri** [comment]
@poppinlp I just started working on.

- [ ] Unit test writing <-
- [ ] Implementation
- [ ] Test

**Comment by RobDesideri** [comment]
@poppinlp my solution is a breaking change.
# Idea
file-changed module should be **instanced**, not used as static object. So you can declare, at instance time, the file path in which store the _**change database**_:
```
var Fc = require('file-changed');
var fc = new Fc('mydb.json');
```
If you not pass argument, it use default file, `_timestamp.json`.
# Benefits
- you can use more than one _change database_ in the project and this can be useful in complex project;
- the _change database_ itself is no longer related to the `node_module` installation folder, so you can delete this folder without worries: the db is stored in your custom path.
# Implementation
You can see the proposed solution in my repo, [develop branch](https://github.com/RobDesideri/file-changed/tree/develop
). 
The key edits is on [these rows](https://github.com/RobDesideri/file-changed/blob/develop/lib/main.js#L19#L29). In addition, now the `export `is the[ class](https://github.com/RobDesideri/file-changed/blob/develop/lib/main.js#L154).
This change also involves a small change to the unit test module: the `fc` variable is instanced [at the beginning](https://github.com/RobDesideri/file-changed/blob/develop/test/test.js#L4#L7), so unit test itself do not undergo any variation.
Let me know what you think.

**Comment by poppinlp** [comment]
Good job. I think that makes sense.
So would you pls send a PR and it'll be released in version 1.0.0 since it's a break change.

**Comment by poppinlp** [comment]
Thanks. I'll publish it tonight (GTM+8) :)
